### ISSUE TITLE
Squeezenet1_1 accuracy minify

### ISSUE THREAD

**Issue by retonym** [intel/torch-xpu-ops#632]
### 🐛 Describe the bug

```

from math import inf
import torch
from torch import tensor, device
import torch.fx as fx
import torch._dynamo
from torch._dynamo.testing import rand_strided
from torch._dynamo.debug_utils import run_fwd_maybe_bwd

import torch._dynamo.config
import torch._inductor.config
import torch._functorch.config
import torch.fx.experimental._config
torch._dynamo.config.automatic_dynamic_shapes = False
torch._dynamo.config.repro_tolerance = 0.001
# torch._dynamo.config.base_dir = '/home/yunfei/code/pytorch/inductor_log/torchbench/squeezenet1_1/training/amp_bf16/14'
torch._inductor.config.fx_graph_cache = True
torch._inductor.config.fallback_random = True
torch._inductor.config.triton.cudagraphs = True

import os
# os.environ["TORCHINDUCTOR_CACHE_DIR"] = "/home/yunfei/code/piece/torch_compile_debug/squeezenet_fail"
# torch._dynamo.config.base_dir = os.environ["TORCHINDUCTOR_CACHE_DIR"]

torch.manual_seed(0)



from torch.nn import *
class Repro(torch.nn.Module):
    def __init__(self):
        super().__init__()
        self.L__self___L__mod___classifier_1 = Conv2d(512, 1000, kernel_size=(1, 1), stride=(1, 1))
        self.L__self___L__mod___classifier_2 = ReLU(inplace=True)
        self.L__self___L__mod___classifier_3 = AdaptiveAvgPool2d(output_size=(1, 1))

        for m in self.modules():
            if isinstance(m, Conv2d):
                if m is self.L__self___L__mod___classifier_1:
                    init.normal_(m.weight, mean=0.0, std=0.01)
                else:
                    init.kaiming_uniform_(m.weight)
                if m.bias is not None:
                    init.constant_(m.bias, 0)



    def forward(self, cat_6):
        _enter_autocast = torch.amp.autocast_mode._enter_autocast('xpu', None, True, None)
        l__mod___classifier_1 = self.L__self___L__mod___classifier_1(cat_6);  cat_6 = None
        l__mod___classifier_2 = self.L__self___L__mod___classifier_2(l__mod___classifier_1);  l__mod___classifier_1 = None
        x_9 = self.L__self___L__mod___classifier_3(l__mod___classifier_2);  l__mod___classifier_2 = None
        _exit_autocast = torch.amp.autocast_mode._exit_autocast(_enter_autocast);  _enter_autocast = None
        return (x_9,)


mod = Repro()

mod = mod.to('xpu')

def load_args(reader):
    buf0 = reader.storage('sb1c3af14cdd479d59b5387919cde8d968a22873f4', 692224, device=device(type='xpu', index=0), dtype_hint=torch.float16)
    reader.tensor(buf0, (4, 512, 13, 13), dtype=torch.float16, requires_grad=True) # cat_6
load_args._version = 0

if __name__ == '__main__':
    from torch._dynamo.repro.after_dynamo import run_repro
    run_repro(mod, load_args, accuracy=True, command='run',
        save_dir='./checkpoints', autocast=False, backend='inductor')

```

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/822214e771fb76b65f701641060e4e8f734708c6
pytorch: 10c7f037fe3271cb3865816c216007ba403f5347 + PRs: 127277,129120
device: PVC 1100, 881, 2024.2

**Comment by retonym** [comment]
The issues should related to conv + relu + adaptive_avgpool_backward pattern.
However, it is really hard to reproduce in unit test. Still working on this problem.

**Comment by chuanqi129** [comment]
Root caused will submit PR later

**Comment by retonym** [comment]
Fix PR: https://github.com/intel/torch-xpu-ops/pull/668

The final solution is to revert pytorch PR: https://github.com/pytorch/pytorch/pull/84541 , after we decide to drop block format solution.

**Comment by mengfei25** [comment]
Failed in release/2.6

**Comment by chuanqi129** [comment]
@mengfei25 check the latest status

**Comment by weishi-deng** [comment]
I check the output numerics for comparison with other backends: the diff ratio between eager and inductor is about 0.00018 on XPU backends, while the tolerance of BF16 is 0.0001.

**Comment by weishi-deng** [comment]
the tolerance PR for pytorch: https://github.com/pytorch/pytorch/pull/156920
